docs/compose-file.md

```diff
@@ -348,6 +348,17 @@ Override the default labeling scheme for each container.
         - label:user:USER
         - label:role:ROLE
 
+### ulimits
+
+Override the default ulimits for a container. You can either use a number
+to set the hard and soft limits, or specify them in a dictionary.
```

Comment:
Wording's a bit strange here. Maybe "You can either specify a single limit as an integer or soft/hard limits as a dictionary".